Packages/com.unity.inputsystem/InputSystem/Actions/InputAction.cs

```diff
@@ -2119,6 +2247,40 @@ public TValue ReadValue<TValue>()
             /// If the currently active control is a <see cref="ButtonControl"/>, the <see cref="ButtonControl.pressPoint"/>
             /// of the button will be taken into account (if set). If there is no custom button press point, the
             /// global <see cref="InputSettings.defaultButtonPressPoint"/> will be used.
+            /// <example>
+            /// <code>
+            /// using UnityEngine;
+            /// using UnityEngine.InputSystem;
+            ///
+            /// public class Example : MonoBehaviour
+            /// {
+            ///     public InputActionReference fire;
+            ///
+            ///     void Awake()
+            ///     {
+            ///         if (fire.action != null)
+            ///         {
+            ///             fire.action.performed += context =>
+            ///             {
+            ///                 // ReadValueAsButton attempts to interpret the value as a button.
+            ///                 bool value = context.ReadValueAsButton();
+            ///                 Debug.Log($"Button state is: {value}");
```

Comment:
same as above comment. Is it usual to use prints in the docs examples?

Reply:
I think it would be enough even to call an empty function like if(value){Jump();} -> Jump() { //insert player jump logic here OR Debug.Log("Jump"); } for example

Reply:
Anyway, I think it's complaining on a high level and is okay to handle like that, too.

Reply:
I think about it an update once CI / package tool issues are resolved

Reply:
there's a few examples of using Debug.Log in our internal docs guide here, so I think this is the best solution:
https://docs.unity.com/internal/content-ops/en-us/documentation/content-model/style-formatting-code-examples